src/libraries/System.Security.Cryptography.Algorithms/src/Resources/Strings.resx

```diff
@@ -312,6 +312,9 @@
   <data name="Cryptography_CipherModeNotSupported" xml:space="preserve">
     <value>The specified CipherMode '{0}' is not supported.</value>
   </data>
+  <data name="Cryptography_CfbCipherNotSupported" xml:space="preserve">
+    <value>CFB mode is not supported for this algorithm.</value>
```

Comment:
Personally, I'd just reuse CipherModeNotSupported and specify CFB.  But this is a tad nicer, I suppose.

Reply:
I am amused that I did not find the existing one and also managed to put my new one directly underneath of it.